reward_score/readme_gen_pred.py

```
@@ -0,0 +1,111 @@
+def compute_score(solution_str, ground_truth, extra_info,
+                  timeout=2.0, default_reward=0.0):
+    """
+    README 生成任务评分：用相似度做规则匹配
+    """
+    # 使用规则匹配评分
+    rule_based_result = rule_based_check(solution_str, ground_truth,
+                                         timeout=timeout,
+                                         default_reward=default_reward)
+
+    # 安全计算 similarity
+    similarity = safe_compute_similarity(solution_str, ground_truth,
```

Ruff (0.12.2)

Undefined name `safe_compute_similarity`

(F821)